scripts/run_mypy_checks.py

```diff
@@ -89,7 +89,6 @@
     'core/domain/html_cleaner_test.py',
     'core/domain/html_validation_service.py',
     'core/domain/html_validation_service_test.py',
-    'core/domain/image_services.py',
```

Comment:
Please, as per issue instructions, also type the test file.

Reply:
@vojtechjelinek I have also typed test file, Can you please verify the changes ?
Thank you